janusgraph-benchmark/src/main/java/org/janusgraph/CQLMultiQueryDropBenchmark.java

```diff
@@ -71,33 +72,27 @@ public WriteConfiguration getConfiguration() {
     public void setUp() throws Exception {
         graph = JanusGraphFactory.open(getConfiguration());
 
-        boolean alreadyAdded = true;
         JanusGraphManagement mgmt = graph.openManagement();
-        if (!mgmt.containsPropertyKey("id")) {
```

Comment:
Wouldn't this run into error, that schema key already exists, when executing this benchmark test multiple times with different parameters?

Reply:
No because after each execution we remove the whole graph via the below method `@TearDown`. You can see all the benchmark tests passed for this PR ([link](https://github.com/porunov/janusgraph/actions/runs/8959195234/job/24604366986)). Particular test results:
```
CQLMultiQueryDropBenchmark.dropVertices                                                                                       N/A           true                          N/A              5000  avgt    5     97.821 ±   21.607  ms/op
CQLMultiQueryDropBenchmark.dropVertices                                                                                       N/A          false                          N/A              5000  avgt    5   1407.599 ±   91.186  ms/op
```

We have one method annotated with `@Setup` and one method annotated with `@TearDown` in this class. Both have a default `Level.Trial` ([doc link](https://javadoc.io/doc/org.openjdk.jmh/jmh-core/0.5.2/org/openjdk/jmh/annotations/Level.html#Trial)). Meaning they will be executed for each run of the benchmark (i.e. once per each unique combination of the parameters. In other words 2 times only).
Thus we will first initialize database then run a benchmark iterations then remove database, re-initialize database and run the second benchmark.

The downside: We re-initialize database with the same schema and data two times (i.e. spend time for the same operations).
The upside: The test executions are mostly fair because underlying storage backend should give the same or similar internal cache hit/miss rate between different runs.

We use this technique for all  other benchmarks (i.e. database re-initialization between different benchmark executions).